engine/src/core/app/application.cpp

```diff
@@ -8,6 +8,7 @@
 #include "events/layer_events.h"
 #include "utils/time/timer.h"
 #include "debug/ui/imgui_layer.h"
+#include <GLFW/glfw3.h>
```

Comment:
why this include is necessary here?

Reply:
Because we set a glfw callback in application currently. It was also included before, just not explicitly. This will have to be decoupled later.

Reply:
yee later if we add more dependencies it will be sure easier to decouple.

Reply:
?? I removed all of the dependencies, this is the only one that was left.